<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CVE Watcher</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon/cve-watcher-icon.ico</normaloff>:/icon/cve-watcher-icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>240</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1024</width>
        <height>768</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>1024</width>
        <height>768</height>
       </size>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
      </property>
      <column>
       <property name="text">
        <string>#</string>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>232</red>
         <green>224</green>
         <blue>205</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Score</string>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>232</red>
         <green>224</green>
         <blue>205</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CVE ID</string>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>232</red>
         <green>224</green>
         <blue>205</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>232</red>
         <green>224</green>
         <blue>205</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Severity</string>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>232</red>
         <green>224</green>
         <blue>205</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date Published</string>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>232</red>
         <green>224</green>
         <blue>205</blue>
        </color>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExport_to_Pdf"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Search</string>
    </property>
    <addaction name="actionCVE_Id"/>
    <addaction name="actionCVSS_Severity"/>
    <addaction name="actionKeyword_Search"/>
   </widget>
   <widget class="QMenu" name="menuHelp_2">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuPoll_Interval">
     <property name="title">
      <string>Poll Interval</string>
     </property>
     <addaction name="action1Hour"/>
     <addaction name="action4Hours"/>
     <addaction name="action8Hours"/>
     <addaction name="action24Hours"/>
    </widget>
    <addaction name="actionReload_Last_24_Hours"/>
    <addaction name="menuPoll_Interval"/>
   </widget>
   <widget class="QMenu" name="menuHelp_3">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
   <addaction name="menuHelp_2"/>
   <addaction name="menuHelp_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExport_to_Pdf">
   <property name="text">
    <string>Export to PDF</string>
   </property>
  </action>
  <action name="actionCVE_Id">
   <property name="text">
    <string>CVE Id</string>
   </property>
  </action>
  <action name="actionCVSS_Severity">
   <property name="text">
    <string>CVSS Severity</string>
   </property>
  </action>
  <action name="actionKeyword_Search">
   <property name="text">
    <string>Keyword Search</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action1Hour">
   <property name="text">
    <string>1 Hour</string>
   </property>
  </action>
  <action name="action4Hours">
   <property name="text">
    <string>4 Hours</string>
   </property>
  </action>
  <action name="action8Hours">
   <property name="text">
    <string>8 Hours</string>
   </property>
  </action>
  <action name="action24Hours">
   <property name="text">
    <string>24 Hours</string>
   </property>
  </action>
  <action name="action24_Hours">
   <property name="text">
    <string>24 Hours</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionReload_Last_24_Hours">
   <property name="text">
    <string>Reload Last 24 Hrs</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
